NEWS.rst

```diff
@@ -5,6 +5,81 @@ User-Visible Changes in Tahoe-LAFS
 ==================================
 
 .. towncrier start line
+Release 1.190 (2024-01-04)
+''''''''''''''''''''''''''
+
+Features
+--------
+
+- Tahoe-LAFS now includes a new "Grid Manager" specification and implementation adding more options to control which storage servers a client will use for uploads. (`#2916 <https://tahoe-lafs.org/trac/tahoe-lafs/ticket/2916>`_)
+- Added support for Python 3.12, and work with Eliot 1.15 (`#3072 <https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3072>`_)
+- The new HTTPS-based storage server is now enabled transparently on the same port as the Foolscap server. This will not have any user-facing impact until the HTTPS storage protocol is supported in clients as well. (`#3902 <https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3902>`_)
```

Comment:
Superseded by later work. It's actually on by default now, I think, unconditionally.

Reply:
There's a later item about this, so maybe just drop this one.